client/src/includes/userbar.ts

```diff
@@ -51,6 +51,7 @@ export class Userbar extends HTMLElement {
       '#wagtail-userbar-template',
     );
     if (!template) return;
+    console.log(this);
```

Comment:
rogue console.log?

Reply:
Yeah😅😅😅
And it was caught by the linter 😅